tests/foreman/cli/test_location.py

```diff
@@ -369,6 +369,23 @@ def test_positive_create_with_medium(self):
         self.assertEqual(loc['installation-media'][0], medium['name'])
 
     @tier1
+    def test_positive_create_with_parent(self):
+        """Create new location with parent location specified
+
+        :id: 49b34733-103a-4fee-818b-6a3386253af1
+
+        :BZ: 1299802
+
+        :expectedresults: Location created successfully and has correct and
+            expected parent location set
+
+        :CaseImportance: High
+        """
+        parent_loc = make_location()
+        loc = make_location({'parent-id': parent_loc['id']})
+        self.assertEqual(loc['parent'], parent_loc['name'])
```

Comment:
The parent is set only by parent-id (as location names can be duplicated), but it display only name,
perhaps we should assert by parent-id also, using the json output
@abalakh what do you think?

Reply:
Regarding json output - i'd rather follow our common approach where it's possible, i don't like the idea to introduce some commands in base format, some in csv, some in json; better stick with 1 format and then fully switch to another if we are asked (imo).
Regarding parent-id - yea, we can cover that, but it will look like some workaround for this particular case, and there's no real need in such workaround - it's typical for our cli to return only name and lots of entities can have duplicated names. We rely on random names generator and so far it works ok for us, the chance of generating exactly the same names is just too low to worry about it and to decrease code simplicity. Long story short, i'm not against such change but i think it's worthless. And your thoughts?

Reply:
For me it's strange that the creation is by id and info is by name that can have different interpretations, not in the case of these tests (that I am sure that they are always unique values in the scope of one tier testing) but in general use case this cannot be true, and can lead to big confusions. 
(The bug verifiers and users was happy with that resolution) so ACK